lists2safebrowsing.py

```diff
@@ -663,10 +663,23 @@ def start_versioning(config, chunknum, shavar_prod_lists_branches):
 
 
 def main():
+    execution_environment = os.getenv("EXECUTION_ENVIRONMENT", "JENKINS")
+    environment = os.getenv("ENVIRONMENT", "stage")
+
     config = configparser.ConfigParser()
-    filename = config.read(["shavar_list_creation.ini"])
+
+    if execution_environment == "GKE":
+        filename = config.read([f"{environment}.ini"])
```

Comment:
```suggestion
        filename = config.read([f"rs_{environment}.ini"])
```
So we can match the file name you use in the repo.

Reply:
Fixed!